assets/js/src/form_editor/components/tutorial.tsx

```diff
@@ -17,6 +18,13 @@ export const Tutorial = () => {
     return null;
   }
 
+  const installedAtDate = new Date(MailPoet.installedAt);
+  const sevenDaysAgo = new Date();
+  sevenDaysAgo.setDate(sevenDaysAgo.getDate() - 7);
+  if (installedAtDate < sevenDaysAgo) {
+    return null; // only show to new users
+  }
+
```

Comment:
I couldn't find this in spec. I'm not sure if that is a good idea because many users may decide to use form editor or even the plugin much later than in 7 days after installing it.
If we don't want to show it to users who have some experience with the form editor maybe we could check if they already have some older form.
What do you think?